### ISSUE TITLE
Dashes or underscores?

### ISSUE THREAD

**Issue by juanpabloaj** [juanpabloaj/pip-init#10]
@msabramo I read your setuptools issue

https://bitbucket.org/pypa/setuptools/issue/307/pep440-code-in-setuptools-821-is-parsing

when I started pip-init package, I had the same question: in the package name of base setup.py file is better a dash or underscore ?

so, when a base setup.py file is created, and if the package name have dashes, is a good idea replace automaticaly the dash to a underscore ?

**Comment by msabramo** [comment]
Yeah I think if I were doing a new package, I would choose underscore over a dash. This PEP 440 problem is one reason. Also, I used a dash for a package at work once and found that our internal package index did not like it. So I would tend to use the underscore instead.

**Comment by ghost** [comment]
underscore
